tests/lib/rules/jsx-max-props-per-line.js

```diff
@@ -32,22 +48,23 @@ ruleTester.run('jsx-max-props-per-line', rule, {
     code: '<App foo />'
   }, {
     code: '<App foo bar />',
-    options: [{maximum: 2}]
+    options: [getConfigs(2)]
```

Comment:
i'm not convinced this is clearer than just repeating each configuration inline. It's much harder for me to evaluate the test cases this way. Can you update this to remove `getConfigs`?